code/game/objects/items/devices/taperecorder.dm

```diff
@@ -203,19 +195,17 @@
 	playsound(src, 'sound/items/taperecorder/taperecorder_play.ogg', 50, FALSE)
 	var/used = mytape.used_capacity	//to stop runtimes when you eject the tape
 	var/max = mytape.max_capacity
-	for(var/i = 1, used < max, sleep(10 * playsleepseconds))
+	for(var/i = 1, used <= max, sleep(playsleepseconds SECONDS)) // <= to let it play if the tape is full
```

Comment:
make it use `SECONDS` in the variable declaration. not here.
and would you put the sleep on the first line of the loop?